<launch>
	<node name="spline_optimization_example" pkg="ewok_optimization" type="spline_optimization_example3" output="screen">
		<rosparam param="x_pos">[2.0, 16.0]</rosparam> 
		<rosparam param="y_pos">[2.0, 2.0]</rosparam> 
		<rosparam param="z_pos">[3.0, 3.0]</rosparam>

		<param name="number_of_target" type="int" value="2"/>
		<param name="target_error" type="double" value="0.1"/>
		<param name="odom_error" type="bool" value="true"/> <!--false-->
		<param name="number_of_points" type="int" value="10"/>
		<param name="dt_value" type="double" value="0.4"/>
	</node>
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find ewok_optimization)/rviz/optimization_example.rviz" />

</launch>
